model/homepage/model.go

```diff
@@ -69,4 +69,5 @@ type Feature struct {
 	Description string `json:"description"`
 	URI         string `json:"uri"`
 	ImageURL    string `json:"image_url"`
+	ImageAlt	string `json:"image_alt"`
```

Comment:
Sorry to be nitpicky, but can you format this so that the fields align with the others in the struct, please? If you have the official `go` extension in VS Code, then every time you save a `.go` file it should do this automatically (and other Go specific formatting) for you: https://code.visualstudio.com/docs/languages/go

Reply:
@rav-pradhan It's not nitpicky! I saw this on the PR and went to reformat this but it was showing that it was aligned correctly...

Reply:
Apologies, needed to uses spaces instead of tab.